pkg/oss/utils.go

```diff
@@ -61,6 +63,24 @@ func GetRAMRoleOption() string {
 	return mntCmdRamRole
 }
 
+// GetRRSAConifg get oidcProviderArn and roleArn
+func GetRRSAConifg(opt *Options, m metadata.MetadataProvider) *mounter.RrsaConfig {
+	saName := fuseServieAccountName
+	if opt.ServiceAccountName != "" {
+		saName = opt.ServiceAccountName
+	}
+
+	if opt.OidcProviderArn != "" && opt.RoleArn != "" {
```

Comment:
If I just set one of `OidcProviderArn` or `RoleArn`, then it is ignored silently. Why not accept them?

Can `RoleArn` replace `RoleName`?

Reply:
Related param-check in nodeserver already existed to ensure "both ARNs are set, or RoleName is set."
I don't think allow RoleArn to replace RoleName is a good idea, it's better to declare both ARNs explicitly if the user want to access OSS across accounts (which is not recommended in fact, and will not be exposed to official documents). Or else,  a simple roleName is enough, which can avoid different accountIDs between roleArn and oidcProviderArn.

Reply:
e.g., user may specify RoleName + OidcProviderArn. Then OidcProviderArn is ignored. ARNs are not for cross account access only.

I would sugguest:
- at least one of RoleName or RoleArn is required. When both specified, raise error on conflict, and accept if they are the same.
- OidcProviderArn is optional

Reply:
For beginner users, RoleName is enough and much easier.
If multiple providers are in need, use both params can also achieve the goal. 
The functions are the same but it can reduce the risk of mismatch between ARNs for the across account case, and facilitate users to adjust parameters. Again, @mowangdk PTAL.